filament/backend/src/opengl/PlatformCocoaGL.mm

```diff
@@ -115,14 +115,20 @@
         glGetIntegerv(GL_VIEWPORT, viewport);
         glGetIntegerv(GL_SCISSOR_BOX, scissor);
 
```

Comment:
Saving and recalling the viewport / scissor state here shouldn't be needed anymore.

Reply:
Okay, added a commit for this.